ui.xml

```diff
@@ -246,7 +246,7 @@
         <Button onClick="toggleBlightUI" textAlignment="MiddleCenter">Toggle Blight UI</Button>
         <Button onClick="toggleScoreUI" textAlignment="MiddleCenter">Toggle Score</Button>
         <Button onClick="togglePowerDrawUI" textAlignment="MiddleCenter">Toggle Power Draw UI</Button>
-        <Button onClick="togglePlayerControls" textAlignment="MiddleCenter" tooltip="Toggles the visibility of the buttons to change player seats or colors.  These are automatically hidden when the game starts, but can be turned back on if desired.">Toggle Seat Controls</Button>
-        <Button onClick="toggleMultihanded" textAlignment="MiddleCenter" tooltip="Toggles whether the visibility the 'Play Spirit' buttons appear for all empty seats or only for those who have chosen a spirit.">Toggle Multihanded</Button>
+        <Button onClick="togglePlayerControls" textAlignment="MiddleCenter" tooltipPosition="Left" tooltip="Toggles the visibility of the buttons to change player seats or colors.  These are automatically hidden when the game starts, but can be turned back on if desired.">Toggle Seat Controls</Button>
```

Comment:
the tooltips are still just one giant line of text, can you break these up into reasonable chunks similar to how i handled the adversary tooltips?

Reply:
Fixed.  This feels like a dirty hack but I suppose it works.  (Has it been tested on non-Windows platforms?)